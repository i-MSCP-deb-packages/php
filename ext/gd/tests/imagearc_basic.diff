001+ 06dfbbf9f77c91cf9e9e1514ec17bb47
001- f18ad8001afefee2e9b8c08d6884425b