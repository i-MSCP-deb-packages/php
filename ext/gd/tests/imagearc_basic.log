
---- EXPECTED OUTPUT
f18ad8001afefee2e9b8c08d6884425b
---- ACTUAL OUTPUT
06dfbbf9f77c91cf9e9e1514ec17bb47
---- FAILED
