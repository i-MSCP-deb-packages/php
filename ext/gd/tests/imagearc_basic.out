06dfbbf9f77c91cf9e9e1514ec17bb47